docs/ogc-protocols/ogc-csw-auth.md

```diff
@@ -3,16 +3,15 @@ The following section will help you understand how to make authenticated request
 
 > :information_desk_person: In order to enquire **auth token**, please [contact us](/classified/contact_us.md).
 
-When making **any** request to our CSW services, the `x-api-key` **header** or `token` **query parameter** must be supplied.<br/>
+When making **any** request to our CSW services, the `token` **query parameter** must be supplied.<br/>
 If token doesn't supplied then the request will get a **403** http response code.
 
 Example for ***GetCapabilities*** request:
 ```curl
```

Comment:
is curl supported use case???

Reply:
i guess it's supported for developers - same as postman